reports/helmcharts/Dockerfile

```diff
@@ -1,8 +1,11 @@
-FROM alpine:3.20.0
+FROM alpine/k8s:1.30.0
 
 ENV PIP_BREAK_SYSTEM_PACKAGES 1
 
 RUN apk update && apk add curl coreutils yq jq openssl github-cli uuidgen bash python3 py3-pip
+RUN rm /usr/lib/python*/EXTERNALLY-MANAGED && python3 -m ensurepip
```

Comment:
You wont need this anymore `rm /usr/lib/python*/EXTERNALLY-MANAGED`
The line `ENV PIP_BREAK_SYSTEM_PACKAGES 1` does the same thing.

Reply:
ooh i see